src/ib_module.F

```diff
@@ -230,6 +232,7 @@ subroutine ib_init(xh,yh,xf,yf,sigma,sigmaf,zs,zh,bndy,kbdy,out3d,  &
         kmaxib = max( kmaxib , kbdy(i,j) )
       enddo
       enddo
+      !$acc update device(kmaxib)
```

Comment:
should this line be moved to after the `MPI_ALLREDUCE` call?

Reply:
Thanks @johnmauff . And unfortunately I need to remove this statement eventually after the discussion yesterday.